pom.xml

```diff
@@ -64,6 +64,13 @@
     <properties>
         <jenkins.version>1.642.3</jenkins.version>
```

Comment:
I'm not bumping any dependencies here - in practice, I don't plan to build/test against any core versions but `2.7.3` (the Blue Ocean base), the latest LTS, and master. But I'm not messing with anything that's already here.